### PR TITLE
Debug and test several issues in iOS bridge.

### PR DESCRIPTION
**Pull request by gavrix**

### CODE REVIEW

File: src/bridge/ExposureNotification.ts [hunk 1]
```diff
@@ -19,6 +19,8 @@ export enum RiskLevel {
 }
 
 export enum Status {
+  // .Undefined is made up status to indicate js client that status hasn't been received from EN framework
+  Undefined = 'undefined',
```
Comment (honkfestival) [inline src/bridge/ExposureNotification.ts:23]: Didn't this get added in #153?

File: ios/CovidShield/ExposureNotification.m [hunk 2]
```diff
@@ -9,30 +9,9 @@
...
-  key.transmissionRiskLevel = 5;
-  
-  NSArray *keys = @[key];
-  completionHandler(keys, nil);
-}
-
-@end
-
-
-typedef ENManagerMock ENManagerImplementation ;
```
Comment (honkfestival) [inline ios/CovidShield/ExposureNotification.m:32]: 👍

Comment (honkfestival) [reply ios/CovidShield/ExposureNotification.m:32]: (a mock might be worth adding back in at a later date once we have a better idea of what the framework does)

Comment (gavrix) [reply ios/CovidShield/ExposureNotification.m:32]: yeah but mock data I tried to come up with doesn't make sense and causes errors when attempt to use by both framework AND backend.

File: src/bridge/ExposureNotification.ts (hunk 1 above)
Comment (gavrix) [reply src/bridge/ExposureNotification.ts:23]: yep, I'll need to rebase once #153 is merged.

**Pull request merged**